tests/certification/bindings/cron/cron_test.go

```diff
@@ -0,0 +1,213 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cron_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/benbjohnson/clock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/components-contrib/bindings/cron"
+	"github.com/dapr/dapr/pkg/runtime"
+	"github.com/dapr/go-sdk/service/common"
+	"github.com/dapr/kit/logger"
+
+	// Certification testing runnables
+	"github.com/dapr/components-contrib/tests/certification/embedded"
+	"github.com/dapr/components-contrib/tests/certification/flow"
+	"github.com/dapr/components-contrib/tests/certification/flow/app"
+	"github.com/dapr/components-contrib/tests/certification/flow/sidecar"
+
+	bindings_loader "github.com/dapr/dapr/pkg/components/bindings"
+	dapr_testing "github.com/dapr/dapr/pkg/testing"
+)
+
+// Test cron trigger with most frequent schedule possible : @every 1s with a deadline
+func TestCronBindingFrequentTrigger(t *testing.T) {
```

Comment:
Why do we need `FrequentTrigger` test? Should it be `TestCronBindingTrigger`? Also based on the test plan, do you want to add other schedules too?

Reply:
adding a test matrix to test various cron formats and schedules